aws/resource_aws_alb_target_group_test.go

```diff
@@ -65,6 +65,7 @@ func TestAccAWSALBTargetGroup_basic(t *testing.T) {
 					resource.TestCheckResourceAttr("aws_alb_target_group.test", "protocol", "HTTPS"),
 					resource.TestCheckResourceAttrSet("aws_alb_target_group.test", "vpc_id"),
 					resource.TestCheckResourceAttr("aws_alb_target_group.test", "deregistration_delay", "200"),
+					resource.TestCheckResourceAttr("aws_alb_target_group.test", "slow_start", "0"),
```

Comment:
We only need to check `slow_start` being equal to `"0"` in the `_basic` resource test and its configuration as the default applies to all configurations unless overridden. Speaking of which, we should have a completely separate acceptance test and configuration that actually sets this to a non-0 value and preferably tries to update it as well so we know that functionality works.